src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadExerciseResource.java

```diff
@@ -193,20 +187,18 @@ private void validateNewOrUpdatedFileUploadExercise(FileUploadExercise fileUploa
     }
 
     /**
-     * GET /courses/:courseId/exercises : get all the exercises.
+     * GET courses/:courseId/file-upload-exercises : get all the exercises.
      *
      * @param courseId the id of the course
      * @return the ResponseEntity with status 200 (OK) and the list of fileUploadExercises in body
      */
-    @GetMapping(value = "/courses/{courseId}/file-upload-exercises")
+    @GetMapping("courses/{courseId}/file-upload-exercises")
     @PreAuthorize("hasRole('TA')")
     public ResponseEntity<List<FileUploadExercise>> getFileUploadExercisesForCourse(@PathVariable Long courseId) {
         log.debug("REST request to get all ProgrammingExercises for the course with id : {}", courseId);
         Course course = courseRepository.findByIdElseThrow(courseId);
         User user = userRepository.getUserWithGroupsAndAuthorities();
-        if (!authCheckService.isAtLeastTeachingAssistantInCourse(course, user)) {
-            return forbidden();
-        }
+        authCheckService.checkHasAtLeastRoleInCourseElseThrow(Role.TEACHING_ASSISTANT, course, user);
```

Comment:
```suggestion
        authCheckService.checkHasAtLeastRoleInCourseElseThrow(Role.TEACHING_ASSISTANT, course, null);
```
and then remove the line above.